src/Input/InputEvents.cpp

```diff
@@ -83,6 +87,8 @@ namespace InputEvents
   static Mode overlay_mode = MODE_DEFAULT;
 
   static unsigned MenuTimeOut = 0;
+  
+  static unsigned LockScreen = 0; // LOCKSCREEN
```

Comment:
Bad code comment, again. What does this variable mean? Add documentation instead of these stupid `LOCKSCREEN` comments!